utils/near-rate-limiter/src/framed_read.rs

```diff
@@ -46,8 +44,14 @@ pin_project! {
 pub struct ThrottleController {
     /// Total count of all messages that are tracked by `RateLimiterHelper`
     num_messages_in_progress: Arc<AtomicUsize>,
+    /// Maximum value of `num_messages_in_progress` recorded since last `consume` call.
+    max_record_num_messages_in_progress: Arc<AtomicUsize>,
     /// Total size of all messages that are tracked by `RateLimiterHelper`
     total_sizeof_messages_in_progress: Arc<AtomicUsize>,
+    /// Stores amount of bandwidth used.
+    bandwidth_used: Arc<AtomicUsize>,
```

Comment:
bandwidth -- is bytes sent and received?

Reply:
received.